airbyte-integrations/connectors/source-hubspot/source_hubspot/api.py

```diff
@@ -280,8 +280,7 @@ def _cast_value(declared_field_types: List, field_name: str, field_value):
         try:
             casted_value = target_type(field_value)
         except ValueError as e:
-            logger.warn(f"Could not cast {field_value} to {target_type}")
-            logger.warn(e)
+            logger.exception(f"Could not cast {field_value} to {target_type}")
```

Comment:
Maybe it's better to add quotes here?
```suggestion
            logger.exception(f"Could not cast `{field_value}` to `{target_type}`")
```

Because if `field_value` is empty string then output will be strange, just 2 spaces:

<img width="287" alt="image" src="https://user-images.githubusercontent.com/19872253/128549587-8e2f651b-3e1b-4a63-a789-71fdd6a36daf.png">

With quotes there will be at least quotes 😅